anomaly_detector/adapters/som_model_adapter.py

```diff
@@ -73,6 +73,7 @@ def train(self, node_map, data, recreate_model=True):
     @latency_logger(name="SomModelAdapter")
     def preprocess(self, config_type, recreate_model):
```

Comment:
This function should check if the config.STORAGE_BACKEND == 'localdir' and if so then we should loop through the list of files processed. Each batch will contain contents of a file.